support-frontend/assets/components/legal/termsPrivacy/sepaTerms.jsx

```diff
@@ -0,0 +1,38 @@
+// @flow
+
+// ----- Imports ----- //
+
+import React from 'react';
+import { type PaymentMethod } from 'helpers/forms/paymentMethods';
+
+// ---- Types ----- //
+
+type PropTypes = {|
+  paymentMethod: PaymentMethod,
+|};
+
+// ----- Component ----- //
+
+function SepaTerms(props: PropTypes) {
+  return (
+    <>
+      <div className="component-terms-privacy">
+        {props.paymentMethod === 'Sepa' ? (
```

Comment:
this condition (and the prop) can be removed, since we're also checking it in the parent container